### ISSUE TITLE
Missing Study icon

### ISSUE THREAD

**Issue by MAttila42** [sameerasw/folder-icons#28]
Hi,
First of all, I love the icons you made!
While downloading some of them I noticed that you are missing a download for the Study icon. This repository's README shows one, but I cannot find it in the PNG or ICO folders, nor on your website (btw the links point to the github.io domain which isn't avaliable anymore).

**Comment by MAttila42** [comment]
It seems like it was deleted in one of the commits. Found it in an earlier version of the repo:
![study](https://github.com/user-attachments/assets/08eb3378-aeb2-4f0d-b165-fb9a282b1688)

**Comment by sameerasw** [comment]
Hey! Thanks for the heads up. I updated the link to my new website I forgot that I moved it. 

The study and a couple other icons were removed because when creating the download/preview [website](https://www.sameerasw.com/icons/categories) I realized that there were a couple of .ico only icons which I made at the start but forgot to save as png. They were causing some issues with the website previews so I removed in order to replace with new ones and I think I missed this one somehow. My bad :)

I'll see if I still have the new icons but not sure since I don't use Windows much anymore ... Will try to re-add those basic ones.

<3
